data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,560 @@
+			
+			label uncle
+			`	She looks wistfully towards the docks. "His uncle is meant to be returning from having gone to pay off the final sum on his debt and wrap up some final business in human space. With the current events, we are both very worried he will not return, though I am doing my best to pretend otherwise."`
+			`	She settles back in her seat with deliberate dignity.`
+				goto leave
+				
+			label peace
+			`	"Thank you, but it is okay, Captain.`
+			label leave
+			`	"Do not trouble yourself over having questioned us. There is no shortcut for processing trauma, and it does the boy no good to hide from the world."`
+			`	She gestures at the screen. "No doubt many people will have many ideas for how to help things soon enough. If it troubles you, lend them a hand. Now, be about your business and farewell."`
+			label walking
+			`	Barely ten minutes later there is already someone on the screen arguing for a range of measures to help with the pirate situation.`
+				decline
+
+
+
+mission "HR: Call for Aid with Sandsnap"
+	name "Join an effort on <planet>"
+	job
+	repeat
+	description "Experienced combat pilots are requested to apply at the spaceport on <destination> for a project."
+	to offer
+		not "hr: sandsnap project"
+	source
+		government "Hai"
+		not planet "Makerplace"
+	destination "Makerplace"
+	to complete
+		has "hr: sandsnap project"
+	on complete
+		payment 1000 150
+
+
+
+mission "Hai Reveal Concern Period: Reactivating Sandsnap"
+	priority
+	name "Transport the Stationkeeper to Sandsnap."
+	description "Travel to <destination> and safely deliver the Stationkeeper to reactivate the station."
+	source "Makerplace"
+	destination "Sandsnap"
+	passengers 1
+	to offer
+		has "hr: allhome raided"
+	on offer
+		conversation
+			`	Upon entering the spaceport you are immediately greeted by a worried hai, "You must be here for the project," he says, not giving you time to respond. He starts walking at a quick pace, with a gesture encouraging you to follow him. "Times are dire, it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the increased activity. Perhaps our brethren north were right in assuming evil intents."`
+			`	He stops himself, "I must be boring you with politics, let us cut to the matter at hand, shall we?"`
+			choice
+				`	(Nod.)`
+					goto border
+				`	"Not at all, I love politics."`
+			`	He seems surprised, "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, demonstrated by how easy those pirate raids were; we were simply not ready for them. But I will do everything to ensure this will not happen again. Mark my words, captain <last>, the Hai will wake up.`
```

Comment:
```suggestion
			`	He seems surprised. "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, demonstrated by how easy those pirate raids were; we were simply not ready for them. But I will do everything to ensure this will not happen again. Mark my words, Captain <last>, the Hai will wake up.`
```
Some Hai are allowed to be sloppy with grammar.